Cargo.lock

```diff
@@ -599,6 +599,12 @@ version = "0.21.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "a4a4ddaa51a5bc52a6948f74c06d20aaaddb71924eab79b8c97a8c556e942d6a"
 
+[[package]]
+name = "base64ct"
```

Comment:
Vetted by skimming (should we also have cargo-vet??)

Reply:
`cargo-vet` is definitely pretty high on my list of tools to add to nearcore :)

In this specific case, I did not vet anything, but the changes are only based on the two Cargo.toml changes, meaning that they are dependencies of `ed25519-dalek`; which I’m assuming (probably wrongly) already vetted the deps. Thank you for having looked at it too :)